iree/compiler/Codegen/SPIRV/KernelConfig.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 The IREE Authors
```

Comment:
Could we mark these as moves?

Reply:
I originally did `git mv`. But I guess it's due to the fact that the majority of the contents are changed; so git treated it as new file.